Cone_spanners_2/include/CGAL/Cone_spanners_2/Plane_scan_tree.h

```diff
@@ -182,14 +182,14 @@ class Plane_scan_tree {
 
     /* End Iterator */
     inline iterator end() {
-        static iterator res;
+        static const iterator res;
```

Comment:
According to that page: http://en.cppreference.com/w/cpp/compiler_support
then the support for the C++11 local thread storage has been added in g++-4.8, clang-3.3, MSVC-14 (aka 2015). When will be ready to drop support for users stuck with previous versions?

For g++ [under Redhat Enterprise Linux, g++-4.8 means RHEL 7][*], the last version. We have users who are stuck with RHEL 6 at the moment. Recently I have even seen a user requesting support for g++ 4.2 !

[*]: https://access.redhat.com/solutions/19458

[The end-of-life of RHEL 6 should be around 2021][wp-RHEL]. Until then, I do not think we can drop the support for C++<11 and we will probably directly jump to support C++>=14. For now, that means that we will have to find workarounds: use tools from Boost when the C++11 version is not usable. That is what we have always done so far. The thread safety is just a rather new topic for CGAL, and around workarounds are not completely ready.

[wp-RHEL]: https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux#Version_history